README.md

```diff
@@ -99,6 +99,9 @@ Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.
 
 Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
 
+## Deployment
+Run `ng build --prod` to build the project in production mode. The artifacts will be stored in the `dist/` directory. This is the folder you need to move on your webspace and configure your Webserver, to route all 404 requests to `index.html`
```

Comment:
@DeeJayBro please add step by step instructions to do angular deployment. For reference you can use this https://angular.io/guide/deployment